src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -38,6 +39,33 @@ const isDrawingLineOrMeasure = computed(() =>
         currentDrawingMode.value
     )
 )
+const selectedEditableFeatures = computed(() => store.state.features.selectedEditableFeatures)
+const selectedLineString = computed(() => {
+    if (selectedEditableFeatures.value && selectedEditableFeatures.value.length > 0) {
+        const selectedFeature = selectedEditableFeatures.value[0]
```

Comment:
As far as I know, no. But the store as a list of editable feature. Perhaps there is a way to do it or some historical reason?
I always select the first item. From the user point of view, it will be the same, since there will be always only one item in the list.